config/jobs/kubernetes/sig-cloud-provider/aws/kops/kops-presubmits.yaml

```diff
@@ -20,7 +20,7 @@ presubmits:
       preset-e2e-platform-aws: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191031-df9cbb4-master
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191103-6816af1-master
```

Comment:
@justinsb I believe the presubmit job is actually defined here: https://github.com/kubernetes/test-infra/blob/0ef1f8e9ffcceffb048205ae26fb72eb61e0ec52/config/jobs/kubernetes/kops/kops-config.yaml#L59-L71

we should delete this duplicate `kops-presubmits.yaml` file.

Reply:
ah nevermind, the file you updated defines an optional job for k/k so we should probably keep it if it is legitimate.